jenkins-integration/scripts/foss_install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+SYSTEST_CONFIG=$1
+SSH_KEYFILE=$2
+
+bundle exec beaker             \
+  --config "$SYSTEST_CONFIG"   \
+  --helper beaker/helper.rb    \
+  --tests beaker/install/foss/ \
+  --preserve-hosts             \
+  --no-color                   \
+  --xml                        \
+  --ntp                        \
+  --root-keys                  \
+  --repo-proxy                 \
+  --debug                      \
+  --keyfile $SSH_KEYFILE
```

Comment:
https://github.com/puppetlabs/puppet-server/blob/master/acceptance/scripts/generic/testrun-full.sh

This is a good pattern to follow for writing these scripts. We should do the `--keyfile` part here and hardcode it